src/cascadia/TerminalApp/ActionArgs.cpp

```diff
@@ -272,4 +273,19 @@ namespace winrt::TerminalApp::implementation
         return L"";
     }
 
+    winrt::hstring ToggleTabSwitcherArgs::GenerateName() const
+    {
+        // If there's an anchor key set, don't generate a name so that
+        // it won't show up in the command palette. Only an unanchored
+        // tab switcher should be able to be toggled from the palette.
+        if (_AnchorKey != Windows::System::VirtualKey::None)
+        {
+            return L"";

```

Comment:
Hmm, it wasn't intentional that you would be able to access anchored mode through the palette (only unanchored should be available) bc chances are the user isn't holding down the anchor as they choose `tab switcher` in the palette, but you said you liked it so now I'm not sure LOL

I believe it shows when you do that because you gave the command a name, and so it avoids `GenerateName()` which is where I blocked the anchored mode command from going in, so I'll have to patch that up.

Reply:
I've punted this to #7179 to track the ability to hide user-named commands from the cmdpal. I think this is minor enough, and maybe other users might enjoy the search-bar-less version more from the cmdpal 😄